webapp/src/utils/errorHandling.ts

```diff
@@ -11,10 +11,10 @@ const getErrorMessage = (
 
     switch (component) {
     case 'SubscribeModal': // Create subscription modal
-        if (errorState.status === 400 && errorState.data.error === pluginConstants.messages.error.subscriptionAlreadyExists) {
-            return errorState.data.error;
+        if (errorState.status === 400 && errorState.data.Error === pluginConstants.messages.error.subscriptionAlreadyExists) {
```

Comment:
Why changed the case of `error`?

Reply:
on debugging I found that it's `Error` and not `error` in the response from RTK

Reply:
Is this because of RTK or Azure API? Do we also need to change this in other plugins?

Reply:
this is from RTK, sure I'll confirm after testing once again on all errors